docs/source/user_guide/model_understanding.ipynb

```diff
@@ -448,8 +448,18 @@
    "source": [
     "from evalml.model_understanding.prediction_explanations import explain_predictions\n",
```

Comment:
@freddyaboulton I see the updates below are for the "Explaining Multiple Predictions" section. Should we also update the "Explaining Individual Predictions" section? I see that example still uses `print(table)`
![Screen Shot 2021-02-03 at 12 32 53 PM](https://user-images.githubusercontent.com/235870/106785906-f4ac1300-661b-11eb-99f3-5ce0f5cd1280.png)

Reply:
I added a section on changing the output format for `explain_prediction` in the "Changing output formats" section!